server/commands/casing.py

```diff
@@ -172,6 +173,18 @@ def ooc_cmd_uncm(client, arg):
             client.send_ooc(
                 f'{id} does not look like a valid ID.')
 
+@mod_only(area_owners=True)
```

Comment:
does this mean you can usurp other cms in the same area - e.g. non-mod cm calls /clear_cm to un-cm a mod

Reply:
yes but they can already do that with /uncm